packages/kbn-test/src/functional_test_runner/lib/config/schema.ts

```diff
@@ -38,6 +38,14 @@ const urlPartsSchema = () =>
       password: Joi.string(),
       pathname: Joi.string().regex(/^\//, 'start with a /'),
       hash: Joi.string().regex(/^\//, 'start with a /'),
+      ssl: Joi.object()
```

Comment:
It should be OneOf([undefined, SslObject]), but I think it's a minor. we should switch FTR to `@kbn/config-schema` 
@spalger

Reply:
`Joi.object().optional()`?

Reply:
created https://github.com/elastic/kibana/issues/69393